webpack.config.prod.js

```diff
@@ -7,7 +7,10 @@ var _package = require('./package.json');
 _.devtool = 'cheap-module-source-map';
 _.entry = './src/index';
 _.plugins = [
-  new webpack.optimize.CommonsChunkPlugin('common.js'),
+  new webpack.optimize.CommonsChunkPlugin({
```

Comment:
@mparke is it ok to go from `common.js` to `commons.js` with no change elsewhere?